walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```

Comment:
Your walkthrough edits came back

Reply:
I honestly don't know how that happened. I've, once again, been having internet connectivity issues and it sometimes undoes work.

Reply:
I also didn't add it to my commit so I don't know how it got here. Git is confusing.

Reply:
It happens :P. That's strange though